doc/tutorials/flashing-duo-s.md

```diff
@@ -0,0 +1,57 @@
+# Set Up
+
+Connect to the board via an ethernet cable.
+Connect to the board via serial.
+To do this, plug in the included USB UART adaptor to your host system, and then plug in the UART cables to the board as described below.
+If you do not plug in to the right pins, you run the risk of frying the board, so please follow these instructions carefully!
+
+You will want to locate the pin on the UART board labeled `+5V`, above that will be `GND`, `RXD`, and `TXD` (shown below).
+We will not be using the `3V3` pin (some of the boards have a cable coming out of `3V3`, ignore it).
+
+![uart-board](../img/uart-board.jpg)
+
+The `+5V` cable will be attached to the second to top pin on the right side of the board, on the column of pins on the outward side, as shown below (black cable).
+The `GND`, `RXD`, and `TXD` will then be plugged into the board below the `+5V`, in order of how they are listed on the UART board, as shown below.
+NOTE - Your cable colors may be different.
+
+![duo](../img/duo.jpg)
+
+Once you have these connected, you will need to serial into the board, as described below.
+
+## Accessing the board via serial (We are assuming you are using Linux)
+
+Make sure you have the `minicom` and `busybox (tftp)` packages installed.
```

Comment:
Probably people should be using the Busybox inside the devcontainer, right? @june-fish just said that might not work but I don't know why not.